zcash_client_backend/src/data_api/wallet/input_selection.rs

```diff
@@ -335,41 +348,41 @@ impl<DbT, ChangeT: ChangeStrategy> GreedyInputSelector<DbT, ChangeT> {
     /// attempting to construct a transaction proposal that requires such an output.
     ///
     /// [`EphemeralBalance::Output`]: crate::fees::EphemeralBalance::Output
-    pub fn new(change_strategy: ChangeT, dust_output_policy: DustOutputPolicy) -> Self {
+    pub fn new() -> Self {
```

Comment:
I think this change is missing in the changelog.

Reply:
It's there: https://github.com/zcash/librustzcash/pull/1579/files#diff-1cc140d33c5912f6e98cc8bd07ee744a81aa7a70ee4becabe24617a5c9608f49R50-R51